release-notes/Cloud/DCP_change_log.md

```diff
@@ -28,6 +28,12 @@ Something went wrong while initializing WebApiEndpointService. A url in the (cus
 
 This fix is included in Cloud Pack 2.8.5.
 
+#### 10 January 2023 - Enhancement - CoreGateway 2.12.0 - Support for new Data Aggregator DxM [ID_34903] [ID_35168] [ID_35217] [ID_35252]
```

Comment:
I'm not sure what the naming conventions for DxMs are? I see here CoreGateway (without space) and SupportAssistant (without space) but Data Aggregator with a space? And SupportAssistant is prefixed with DataMiner and the others are not? Shouldn't all DxMs have consistent naming?

Reply:
Indeed, there is an inconsistency here. In principle we should only use the spelling without space when referring to the literal file name, whereas we should use the spelling with space when referring to the module in general (similar to how we mention "Log Collector" even though the corresponding file name is "SL_LogCollector").